.github/workflows/build.yaml

```diff
@@ -12,6 +12,7 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
This can be removed. But right now it is hard to debug actions as all jobs exit when one fails.